protocol/avdl/common.avdl

```diff
@@ -104,12 +118,15 @@ protocol Common {
       UID uid;
       string username;
 
-      // This is a misnomer; it's actually all keys, including PGP; let's deprecate
-      // this field in a few versions
+      // deviceKeys is a list of active device keys.  It does
+      // not include PGP keys.
       array<PublicKey> deviceKeys;
 
-      // A duplicate of deviceKeys above, but with a better name
-      array<PublicKey> keys;
+      // revokedKeys is a list of revoked device keys.
+      array<RevokedKey> revokedKeys;
```

Comment:
If `revokedKeys` needs to leave out revoked PGP keys, should we maybe name it `revokedDeviceKeys`?